app/uk/gov/hmrc/plasticpackagingtax/registration/models/genericregistration/Partner.scala

```diff
@@ -27,7 +27,9 @@ case class Partner(
   soleTraderDetails: Option[SoleTraderDetails] = None,
   incorporationDetails: Option[IncorporationDetails] = None,
   partnershipDetails: Option[PartnershipDetails] = None,
-  contactDetails: Option[PartnerContactDetails] = None
+  contactDetails: Option[PartnerContactDetails] = None,
+  organisationName: Option[String] =
+    None // TODO for other partners flow; change after GRS flow results are available
```

Comment:
I've added a val for the name which is computed using type and details from the appropriate details field. See https://github.com/hmrc/plastic-packaging-tax-registration-frontend/pull/318/files#diff-30e695684abbec2bec62ca9ac2ea92cb496e52a1bd967b89d74999babd2cfd7d